source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -286,6 +288,36 @@ void AuthenticatorImpl::handleGoodJwt(bool cache_hit) {
     }
   }
 
+  // Copy JWT Claim to Header
+  if (provider.claim_to_headers_size() != 0) {
+    for (const auto& header_and_claim : provider.claim_to_headers()) {
+      if (!header_and_claim.claim_name().empty() && !header_and_claim.header_name().empty()) {
+        StructUtils payload_getter(jwt_->payload_pb_);
+        const ::google::protobuf::Value* found;
+        google::jwt_verify::StructUtils::FindResult claim_value =
```

Comment:
move these codes for each header/claim into a separate function?

Reply:
sure